sig-windows/charter.md

```diff
@@ -14,13 +14,14 @@ Windows specific implementation.
 
 #### Code, Binaries and Services
 
-- Windows specific code in all parts of the codebase.
+- Windows specific code in all parts of the codebase
 - Testing of Windows specific features and clusters
 
 #### Cross-cutting and Externally Facing Processes
 
 - Work with other SIGs on areas where Windows and Linux (and possibly other OSes in the future) deviate from one another in terms of functionality.
-
+- Providing community guidance for external tools and paradigms necessary for production windows k8s administration
```

Comment:
`Windows` should be uppercase like the rest of the doc.

Reply:
agreed